src/checkout/create-checkout-service.ts

```diff
@@ -43,9 +43,16 @@ export default function createCheckoutService(options?: CheckoutServiceOptions):
         getDefaultLogger().warn('Note that the development build is not optimized. To create a production build, set process.env.NODE_ENV to `production`.');
     }
 
+    const config: ConfigState = {
+        meta: {
+            externalSource: options && options.externalSource,
```

Comment:
Don't think we should set a default value in the sdk because custom checkout that did not set a value will be getting this default value unintentionally.